src/systems/particle_emitter/ParticleEmitter.cc

```diff
@@ -258,6 +258,15 @@ void ParticleEmitter::Configure(const Entity &_entity,
         absolutePath);
   }
 
+  // particle scatter ratio
+  if (_sdf->HasElement("particle_scatter_ratio"))
+  {
+    auto data = this->dataPtr->emitter.mutable_header()->add_data();
```

Comment:
I'm assuming that we are storing the scatter ratio data in the header since there's no dedicated scatter field in the message type, right? Should we add a TODO note or something similar to update the message type and the storage of this value in the next Ignition release?

Reply:
added todo note. 0718e8980691152abd258a8370b1728ca73f17ca